inc/Engine/Media/Fonts/Controller/Filesystem.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WP_Rocket\Engine\Media\Fonts\Controller;
+
+class Filesystem {
```

Comment:
is there any possibility of re-factoring the filesystem class in Used CSS and  this one for common parts?